src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(...args) {
+    if (args.length === 0) {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    }
+    sum += args.reduce((acc, val) => acc + val, 0);
```

Comment:
The use of `reduce` here is correct for summing the values in `args`. However, ensure that all elements in `args` are numbers to avoid unexpected behavior. Since the task specifies not to check if the argument is a number, this is just a cautionary note.